workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -29,7 +29,14 @@ private static void BuildConfigMenu()
         private static void LaunchMenu()
         {
             Debug.Log("Launching SpatialOS locally...");
-            EditorApplication.delayCall += Launch;
+            EditorApplication.delayCall += LaunchSpatialLocally;
+        }
+
+        [MenuItem("SpatialOS/Launch additional client")]
```

Comment:
Nit: `Launch standalone client` - the user may not have a client running yet

Reply:
Similar changes to logs/function names throughout

Reply:
Definitely makes more sense as `standalone`.